testsuite/integration-arquillian/HOW-TO-RUN.md

```diff
@@ -795,17 +795,34 @@ land by adjusting load balancer configuration (e.g. to direct the traffic to onl
 
 For an example of a test, see [org.keycloak.testsuite.crossdc.ActionTokenCrossDCTest](tests/base/src/test/java/org/keycloak/testsuite/crossdc/ActionTokenCrossDCTest.java).
 
-The cross DC requires setting a profile specifying used cache server by specifying
-`cache-server-infinispan` or `cache-server-jdg` profile in maven.
+The cross DC requires setting a profile specifying the used cache server.
+Use `cache-server-infinispan` Maven profile for Infinispan 10 or higher, or `cache-server-legacy-infinispan` profile for Infinispan 9 and lower.
+Use `cache-server-datagrid` Maven profile for Datagrid 8 or higher, or `cache-server-legacy-datagrid` profile for Datagrid 7 and lower.
 
-Since JDG does not distribute `infinispan-server` zip artifact anymore, for `cache-server-jdg` profile it is
-necessary to download the artifact and install it to local Maven repository. For JDG 7.3.8, the command is the following:
+To specify a custom Java platform to run the cache server it is possible to set parameter: `-Dcache.server.java.home=<PATH_TO_JDK>`.
+
+### Cache Authentication
+
+With WildFLy/EAP based auth server option it is possible to enable authentication for the HotRod protocol by enabling profile `cache-auth`.
+
+It is possible to specify additional parameters:
+- `-Dhotrod.sasl.mechanism`: SASL mechanism used by the hotrod protocol. Default value is `DIGEST-MD5`.
+- `-Dkeycloak.connectionsInfinispan.hotrodProtocolVersion`: Version of the hotrod protocol.
+
+Example: `-Pauth-server-wildfly,cache-server-infinispan,cache-auth -Dhotrod.sasl.mechanism=SCRAM-SHA-512`
+
+### Data Grid
+
+Since Datagrid does not distribute `infinispan-server` zip artifact, for `cache-server-datagrid` profile it is
+necessary to download the artifact and install it to local Maven repository. For Red Hat Data Grid 8 and above, the command is the following:
 
     mvn install:install-file \
-    -DgroupId=org.infinispan.server -DartifactId=infinispan-server -Dpackaging=zip -Dclassifier=bin -DgeneratePom=true \
-    -Dversion=9.4.21.Final-redhat-00002 -Dfile=jboss-datagrid-7.3.8-server.zip
+    -DgroupId=com.redhat -DartifactId=datagrid -Dpackaging=zip -Dclassifier=bin -DgeneratePom=true \
+    -Dversion=${DATAGRID_VERSION} -Dfile=redhat-datagrid-${DATAGRID_VERSION}-server.zip
+
+For Data Grid 7 and older the use: `-Dfile=jboss-datagrid-${DATAGRID_VERSION}-server.zip`.
```

Comment:
Just nitpick, I would drop `the` here. IOW so it reads: `For Data Grid 7 and older use: ...`

Reply:
Addressed in `40d2230`.